### DISCUSSION TITLE
Date Suggestion/Selection

### DISCUSSION THREAD

**Discussion by woessmich** [jonaswinkler/paperless-ng#1535] [Feature Requests]
While other fields do have some suggested values, this does not happen for the date field.
Sometimes document have several dates and only one does apply.
It would be nice if they would be shown as suggestions, as e.g. docspell does.

**Comment by maeries** [comment]
looks like a duplicate of #1177

**Reply by woessmich** [reply]
Yes, it is. Thanks for having a look at the suggestion.
